decidim-admin/app/commands/decidim/admin/create_participatory_space_private_user.rb

```diff
@@ -47,7 +47,7 @@ def create_private_user
       end
 
       def create_or_invite_user
-        @user ||= existing_user || new_user
+        @create_or_invite_user ||= existing_user || new_user
```

Comment:
This seems to be copied from another command. The `@user` variable was set here so that the `attr_reader` works. Now the `user` method returns nil since this change.

Reply:
@mrcasals  then how can we solve this ? `Naming/MemoizedInstanceVariableName: Memoized variable @user does not match method name create_or_invite_user. Use @ create_or_invite_user instead.`

Reply:
Make the method be called `def user` and remove the `attr_reader`